src/visualization/manipulator.cc

```diff
@@ -100,14 +101,13 @@ QString TrackballManipulator::manipulatorName() {
 
 bool TrackballManipulator::eventFilter(QObject* watched, QEvent* event) {
   QWidget* widget = dynamic_cast<QWidget*>(watched);
-  if (widget == nullptr) {

```

Comment:
I thought that Google style guide prefers `!widget`, but seems that's true only for Python style guide. Keep it or revert?